tests/framework/installer/ceph_manifests_v1.5.go

```diff
@@ -964,6 +964,12 @@ metadata:
 ---
 apiVersion: v1
 kind: ServiceAccount
+metadata:
```

Comment:
Actually, don't set this in the 1.5 test manifests, please revert this file. We want the upgrade to create it as part of the new common.yaml. I believe we'll need a fix to the upgrade test, let's talk about this tomorrow after I merge #7390.

Reply:
With #7390 I added a fix to the upgrade test that should make this scenario already work. After you rebase and fix the merge conflicts the tests should pass.

Reply:
Yes @travisn , you have upgraded the manifests file and manifests_v1.5  with `common.yaml` settings so all the **resources** must be taken from `common.yaml` file, 
I have Rebase my commit but now the tests are failing, I got one point  on #7390 that maybe causing error.

The `common.yaml` in [ceph_manifests.go](https://github.com/rook/rook/blob/master/tests/framework/installer/ceph_manifests.go) file is called by [m.settings.readManifest](https://github.com/rook/rook/blob/3bbc358d321ae3b9dc9ad3dca69eaaabe2d9ced5/tests/framework/installer/ceph_manifests.go#L92) and in its definition [readManifestFromGithubWithClusterNamespace](https://github.com/rook/rook/blob/3bbc358d321ae3b9dc9ad3dca69eaaabe2d9ced5/tests/framework/installer/ceph_settings.go#L82) is called and this function is calling [raw content from github](	https://github.com/rook/rook/blob/3bbc358d321ae3b9dc9ad3dca69eaaabe2d9ced5/tests/framework/installer/ceph_settings.go#L87) so that the issue of calling the raw content from web.

As in the **commit** if their any changes in `common.yaml` file so these are not reflected by this **url** of raw content from github. 
So the integration tests will fail in such cases.

I think linking the file by giving the **path** to the file or  importing the package will help instead of mapping it with the web version.

Reply:
are you seeing the issue being fixed in - https://github.com/rook/rook/pull/7451

Reply:
Hy @sp98 , sorry  but I couldn't relate this to 7451 issue
But what I probaby think [ceph_settings.go](https://github.com/rook/rook/blob/3bbc358d321ae3b9dc9ad3dca69eaaabe2d9ced5/tests/framework/installer/ceph_settings.go) file should be modified and should edit the content of [ line raw content from github](	https://github.com/rook/rook/blob/3bbc358d321ae3b9dc9ad3dca69eaaabe2d9ced5/tests/framework/installer/ceph_settings.go#L87).